### ISSUE TITLE
Minify JavaScript and disable logging as part of build procedure

### ISSUE THREAD

**Issue by mzedeler** [Notalib/LYT#334]
Write a new task for the `Cakefile` called `prod` that runs `app` first and then the following:
- Disables logging.
- Minifies the resulting JavaScript.

Issue #333 must be completed prior to this one.

_Update_: I have changed the behaviour. Try running `cake` to see the list of switches. The `prod` mode is equivalent to `cake -m app` and the old `app` mode is equivalent to `cake -d app`.

Note that using the `m` switch requires `uglifyjs2`. Install it by writing `npm -g install uglify-js2`.
